.pipelines/templates/.builder-release-template.yaml

```diff
@@ -58,6 +58,7 @@ steps:
       if [[ ${OS_VERSION} == "22.04" && ${ARCHITECTURE,,} == "arm64" ]]; then SKU_NAME="${SKU_NAME}arm64"; fi && \
       if [[ ${OS_VERSION} == "18.04" && ${ENABLE_FIPS,,} == "true" ]]; then SKU_NAME="${SKU_NAME}fips"; fi && \
       if [[ ${OS_VERSION} == "20.04" && ${ENABLE_FIPS,,} == "true" ]]; then SKU_NAME="${SKU_NAME}fips"; fi && \
+      if [[ ${OS_VERSION} == "22.04" && ${ENABLE_FIPS,,} == "true" ]]; then SKU_NAME="${SKU_NAME}fips"; fi && \
```

Comment:
nit: can we combine L59/60/61, seems redundant to just keep adding as we onboard new Ubuntu versions.

if [[ 1804 or 2004 or 2204 ] and [ fips ]]; then SKUNAME=${SKUNAME}fips; fi

please treat this as pseudocode and not actual bash :)